src/NzbDrone.Core/DecisionEngine/Specifications/ReleaseRestrictionsSpecification.cs

```diff
@@ -30,37 +30,45 @@ public virtual Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase
             _logger.Debug("Checking if release meets restrictions: {0}", subject);
 
             var title = subject.Release.Title;
-            var restrictions = _releaseProfileService.AllForTags(subject.Series.Tags);
+            List<ReleaseProfile> restrictions = _releaseProfileService.AllForTags(subject.Series.Tags);
```

Comment:
I noticed `var` is the standard here rather than explicit typing. I changed it for personal readability as I was trying to figure out what I was working with. If you want it changed back to var I can do that.

Reply:
Yes, the variable name could've been better, maybe 'profiles'.